GitUI/UserControls/RevisionGrid.cs

```diff
@@ -3644,7 +3648,7 @@ private void CompareToBranchToolStripMenuItem_Click(object sender, EventArgs e)
                 if (form.ShowDialog(this) == DialogResult.OK)
                 {
                     var baseCommit = Module.RevParse(form.BranchName);
-                    UICommands.ShowFormDiff(IsFirstParentValid(), baseCommit, headCommit.ObjectId,
+                    UICommands.PerformDiff(IsFirstParentValid(), baseCommit, headCommit.ObjectId,
```

Comment:
Should this really be changed?

Reply:
I'm not sure the issue here. I presume we would want to be able to perform a directory diff from "Compare to branch" popup dialog, would we not?

Or is the comment in regards to the general renaming from ShowFormDiff to PerformDiff. That was a decision I made to clearly distiguish that this may not show a GitExtensions form. PerformDiff encompases the logic that either launches the dir-diff tool directly, or shows a GitExtensions form.